pkg/api/types.go

```diff
@@ -452,6 +452,11 @@ type StepConfiguration struct {
 type InputImageTagStepConfiguration struct {
 	BaseImage ImageStreamTagReference         `json:"base_image"`
 	To        PipelineImageStreamTagReference `json:"to,omitempty"`
+
+	// The Source denotes what caused this image to get loaded. In the event
+	// that the image was loaded as a result of a test step, the source will
+	// contain the name of the test.
+	Source string `json:"source,omitempty"`
```

Comment:
IMO this should not be an externally-visible (exported, JSON-serialized) field, since it's set internally